components/brave_wallet_ui/utils/asset-utils.ts

```diff
@@ -14,20 +14,47 @@ export const getUniqueAssets = (assets: BraveWallet.BlockchainToken[]) => {
   })
 }
 
-export const isSelectedAssetInAssetOptions = (selectedAsset: BraveWallet.BlockchainToken, assetOptions: BraveWallet.BlockchainToken[]) => {
+export const isSelectedAssetInAssetOptions = (
+  selectedAsset: BraveWallet.BlockchainToken,
+  assetOptions: BraveWallet.BlockchainToken[]
+) => {
   return assetOptions.findIndex(asset => {
     return asset.contractAddress.toLowerCase() === selectedAsset?.contractAddress.toLowerCase() &&
       asset.chainId === selectedAsset.chainId &&
       asset.symbol.toLowerCase() === selectedAsset.symbol.toLowerCase()
   }) !== -1
 }
 
+export const getWyreAssetSymbol = (asset: BraveWallet.BlockchainToken) => {
+  if (
+    !asset.contractAddress || // gas coins ok
+    asset.chainId === BraveWallet.MAINNET_CHAIN_ID // ETH-ERC coins ok
+  ) {
+    return asset.symbol
+  }
+
+  // format non-ethereum EVM token symbols for Wyre
+  const prefix = getWyreNetworkPrefix(asset.chainId)
+  return prefix ? `${prefix}${asset.symbol.toUpperCase()}` : asset.symbol
+}
+
 export const getRampAssetSymbol = (asset: BraveWallet.BlockchainToken) => {
   if (asset.symbol.toUpperCase() === 'BAT' && asset.chainId === BraveWallet.MAINNET_CHAIN_ID) {
     // BAT is the only token on Ethereum Mainnet with a prefix on Ramp.Network
     return 'ETH_BAT'
   }
 
+  if (asset.chainId === BraveWallet.AVALANCHE_MAINNET_CHAIN_ID && asset.contractAddress === '') {
+    return asset.symbol // AVAX native token has no prefix
+  }
+
   const rampNetworkPrefix = getRampNetworkPrefix(asset.chainId)
   return rampNetworkPrefix !== '' ? `${rampNetworkPrefix}_${asset.symbol.toUpperCase()}` : asset.symbol
 }
+
+export const addLogoToToken = (token: BraveWallet.BlockchainToken) => {
+  return {
+    ...token,
+    logo: `chrome://erc-token-images/${token.logo}`
+  }
+}
```

Comment:
use internal logos

Reply:
@kdenhartog check if chrome-untrusted is necessary

Reply:
@josheleonard yeah let's use chrome-untrusted for this. It looks like we've migrated other PRs over to this approach as well for image rendering in the wallet

Reply:
Will we need to us the untrusted iframe for ALL image file extensions?

Also, can I use a query string in the iframe that contains the remote image URL, so that I do not need to send `postMessage`s to each iframe?

Reply:
@kdenhartog 

Standalone PR for this here: https://github.com/brave/brave-core/pull/14535
Standalone Security review: https://github.com/brave/security/issues/970

Reply:
Thanks for breaking that one out